modules/platform/nuxeo-preview-core/src/test/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,4 @@
+Manifest-Version: 1.0
```

Comment:
Don't need this file, just deploy your test contrib with `org.nuxeo.ecm.platform.preview:test-dummy-blob-provider.xml`

Reply:
👍